consumer.go

```diff
@@ -610,12 +612,7 @@ func (r *Consumer) ConnectToNSQDs(addresses []AddrPartInfo) error {
 	return nil
 }
 
-// ConnectToNSQD takes a nsqd address to connect directly to.
-//
-// It is recommended to use ConnectToNSQLookupd so that topics are discovered
-// automatically.  This method is useful when you want to connect to a single, local,
-// instance.
-func (r *Consumer) ConnectToNSQD(addr string, part int) error {
+func (r *Consumer) ConnectToNSQDWithExt(addr string, part int, ext bool) error {
```

Comment:
If the lookup query returned the extend_support, I think we should set it to topic consumer. So we do not a new `ConnectToNSQDWithExt` method.